### ISSUE TITLE
Not able to point to v2 helm chart

### ISSUE THREAD

**Issue by guru1602** [grafana/k8s-monitoring-helm#814]
Hi Pete,  
We use k8s helm chart which is v1 (noticed it recently due yo release of v2) to collect metrics; wanted to add in application observability and noticed there is a new chart available which is completely different to what we have.  After watching Monday office hour video was trying to use v2 chart so that we can migrate from v1 before I configure application observability but having an issue while installing the new chart. 

Basically it's referencing to v1 chart rather than v2, not sure if anyone is facing this. 

helm repo add grafana https://grafana.github.io/helm-charts
helm repo update
helm upgrade --install k8smon grafana/k8s-monitoring --namespace "monitoring-test" --create-namespace --values values.yaml  

![Image](https://github.com/user-attachments/assets/2f1d545e-4162-4776-9f32-36aeac735647)

Any suggestion here is appreciated!   

Thanks

**Comment by petewall** [comment]
You should be able to install it now with:

```bash
% helm install --devel k8smon grafana/k8s-monitoring --version 2.0.0-rc.1 -f values.yaml
```
